openmetadata-ui/src/main/resources/ui/src/enums/search.enum.ts

```diff
@@ -42,3 +44,35 @@ export enum SearchIndex {
   DASHBOARD_DATA_MODEL = 'dashboard_data_model_search_index',
   DATA_PRODUCT = 'data_product_search_index',
 }
+
+export const ENTITY_TO_SEARCH_INDEX_MAP: Record<string, SearchIndex> =
```

Comment:
We have `getEntityTypeSearchIndexMapping` in  SearchClassBase so can we use that, if needed please update the list in that method.